File: xide-standard-plugins/xide-logs-plugin/src/main/java/org/xast/xide/logs_plugin/LogsPlugin.java
package org.xast.xide.logs_plugin;

import org.xast.xide.core.event.EventBus;
import org.xast.xide.core.plugin.bottom.BottomPanelPlugin;
import org.xast.xide.core.plugin.bottom.BottomPanelView;

public class LogsPlugin implements BottomPanelPlugin {
    @Override
    public String tabName() {
        return "Logs";
    }

    @Override
    public BottomPanelView view(EventBus eventBus) {
        return new LogsView();
    }
}

File: xide-standard-plugins/xide-logs-plugin/src/main/scala/org/xast/xide/logs_plugin/LogsPlugin.scala
package org.xast.xide.logs_plugin

import org.xast.xide.core.event.EventBus
import org.xast.xide.core.plugin.bottom.BottomPanelPlugin
import org.xast.xide.core.plugin.bottom.BottomPanelView

class LogsPlugin extends BottomPanelPlugin:
    
    override def tabName: String = "Logs"

    override def view(eventBus: EventBus): BottomPanelView = new LogsView()